From bf2b36e087de73c6032a278dfb79a4f652beb78a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Le=20Goffic?= <clement.legoffic@foss.st.com>
Date: Mon, 28 Apr 2025 15:25:41 +0200
Subject: [PATCH 26/95] dt-bindings: usb: stm32: fix dt_binding_check errors
 for USBH and DWC3
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove const from compatible enum.
Add extra space at the beginning of the comment.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/linux.git d2627d73f96adea20973b10ce01b020f28777733 dt-bindings: usb: stm32: fix dt_binding_check errors for USBH and DWC3]
Change-Id: I29bc7b94744846c7307922aaaef3eef61ce1b5e4
Signed-off-by: Clément Le Goffic <clement.legoffic@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/455184
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 Documentation/devicetree/bindings/usb/stm32,dwc3.yaml | 2 +-
 Documentation/devicetree/bindings/usb/stm32,usbh.yaml | 6 ++++--
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/usb/stm32,dwc3.yaml b/Documentation/devicetree/bindings/usb/stm32,dwc3.yaml
index e08390426b1ae..cddc6ef36a3f4 100644
--- a/Documentation/devicetree/bindings/usb/stm32,dwc3.yaml
+++ b/Documentation/devicetree/bindings/usb/stm32,dwc3.yaml
@@ -26,7 +26,7 @@ properties:
   ranges: true
 
   st,syscfg:
-    $ref: /schemas/types.yaml#/definitions/phandle-array #(arm/stm32/st,stm32-syscon.yaml#)
+    $ref: /schemas/types.yaml#/definitions/phandle-array # (arm/stm32/st,stm32-syscon.yaml#)
     description: <Phandle to the STM32-MPU System Controller DT node> <offset-for-usr3drcr-reg>
 
   st,vbusen-active-low:
diff --git a/Documentation/devicetree/bindings/usb/stm32,usbh.yaml b/Documentation/devicetree/bindings/usb/stm32,usbh.yaml
index 22896673c0789..2143b85379031 100644
--- a/Documentation/devicetree/bindings/usb/stm32,usbh.yaml
+++ b/Documentation/devicetree/bindings/usb/stm32,usbh.yaml
@@ -11,7 +11,9 @@ maintainers:
 
 properties:
   compatible:
-    const: st,stm32mp25-usbh
+    enum:
+      - st,stm32mp21-usbh
+      - st,stm32mp25-usbh
 
   "#address-cells":
     const: 1
@@ -22,7 +24,7 @@ properties:
   ranges: true
 
   st,syscfg:
-    $ref: /schemas/types.yaml#/definitions/phandle-array #(arm/stm32/st,stm32-syscon.yaml#)
+    $ref: /schemas/types.yaml#/definitions/phandle-array # (arm/stm32/st,stm32-syscon.yaml#)
     description: Phandle to the STM32-MPU System Controller DT node
                  <syscon-handle usbhcr-offset>
 
-- 
2.51.0

